src/main/resources/db/migration/postgresql/V1_1__CREATE_MESSAGE_LOG_TABLE.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS message_log
+(
+    message_id          VARCHAR(100)                PRIMARY KEY,
```

Comment:
Can be up to 100 character (https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-queue-message-identifiers.html). My experience this has typically been a UUID